test/parallel/test-vm-context.js

```diff
@@ -56,10 +56,11 @@ try {
 } catch (e) {
   gh1140Exception = e;
   assert.ok(/expected-filename/.test(e.stack),
-            'expected appearance of filename in Error stack');
+            `expected appearance of filename in Error stack: ${e.stack}`);
 }
-assert.ok(gh1140Exception,
-          'expected exception from runInContext signature test');
+let eMessage = 'expected exception from ';
+eMessage += `runInContext signature test: ${gh1140Exception}`;
+assert.ok(gh1140Exception, eMessage);
```

Comment:
It would be nicer to just use string concatenation instead of adding a separated variable.